harper-core/src/spell/full_dictionary.rs

```diff
@@ -116,6 +128,13 @@ impl FullDictionary {
 
         word_len_starts
     }
+
+    pub fn get_word(&self, index: usize) -> &CharString {
```

Comment:
I'm not a huge fan of making the underlying indices of the word list part of the API for this struct. Is there a way to avoid this?

Reply:
This is because the `FstDictionary` needs to index into `FullDictionary`. I guess we can make it `pub(super)`?

Reply:
It did not need to. I fixed it.